HelpMyStreetFE/HelpMyStreetFE/Views/Shared/_AccountNavigation.cshtml

```diff
@@ -12,10 +12,11 @@
                 <a href="/account/streets" class="@(Model.CurrentPage == MenuPage.MyStreets ? "selected" : string.Empty)">My Streets</a>
             </li>
         }
-        @if (Model.UserDetails.IsVerified) { 
-            <li class="account__nav__item">
-                <a href="/account/comingSoon" class="@(Model.CurrentPage == MenuPage.ComingSoon ? "selected" : string.Empty)">Coming Soon...</a>
-            </li>
-         }
+        <li class="account__nav__item">
```

Comment:
Should we only display these links to verified users?

Reply:
That's what I initially had but the ACs show different text for verified/unverified on users on the pages